latest commit "Add missing translation in Javascript export dialog" compiling errors

**Godot version:**
<!-- If thirdparty or self-compiled, specify the build date or commit hash. -->


**OS/device including version:**
<!-- If graphics related, specify also GPU model and drivers. -->


**Issue description:**
<!-- What happened, and what was expected. -->
d78335d87f4441c2bbe7e1c11642fbf19ae95c12

platform/javascript/export/export.cpp:280:161: error: expected ')' before ';' token
 ->show_warning(TTR("Could not read boot splash image file:") + "\n" + splash_path + "\n" + TTR("Using default boot splash image.")--->>)<<---;

miss ")" :D

@akien-mga

**Steps to reproduce:**


**Minimal reproduction project:**
<!-- Optional but greatly speeds up debugging. You can drag and drop a zip archive to upload it. -->


- [ ] I searched the existing [GitHub issues](https://github.com/godotengine/godot/issues?utf8=%E2%9C%93&q=is%3Aissue+) for potential duplicates.

```platform/javascript/export/export.cpp: In member function 'virtual Error EditorExportPlatformJavaScript::export_project(const Ref<EditorExportPreset>&, bool, const String&, int)':
platform/javascript/export/export.cpp:280:161: error: expected ')' before ';' token
    EditorNode::get_singleton()->show_warning(TTR("Could not read boot splash image file:") + "\n" + splash_path + "\n" + TTR("Using default boot splash image.");
                                                                                                                                                                 ^
scons: *** [platform/javascript/export/export.x11.tools.64.o] Error 1
scons: building terminated because of errors.
```

--- a/platform/javascript/export/export.cpp
+++ b/platform/javascript/export/export.cpp
@@ -277,7 +277,7 @@ Error EditorExportPlatformJavaScript::export_project(const Ref<EditorExportPrese
 		splash.instance();
 		Error err = splash->load(splash_path);
 		if (err) {
-			EditorNode::get_singleton()->show_warning(TTR("Could not read boot splash image file:") + "\n" + splash_path + "\n" + TTR("Using default boot splash image.");
+			EditorNode::get_singleton()->show_warning(TTR("Could not read boot splash image file:") + "\n" + splash_path + "\n" + TTR("Using default boot splash image."));
 			splash.unref();
 		}
 	}

